src/warehouses/components/WarehouseDetailsPage/WarehouseDetailsPage.tsx

```diff
@@ -85,6 +93,10 @@ const WarehouseDetailsPage: React.FC<WarehouseDetailsPageProps> = ({
           countryChoices
         );
 
+        const [isPrivate, setIsPrivate] = React.useState<string>(
```

Comment:
No need for explicit typing here

```suggestion
        const [isPrivate, setIsPrivate] = React.useState(
```

Reply:
Actually it's the same as `data.isPrivate` so need to keep this hook at all

Reply:
Agree, I have thrown it away.